backend/auth/jwt_manager.py

```
@@ -0,0 +1,346 @@
+- Secure token management
+- User authentication decorators
+"""
+
+import jwt
+import secrets
+import logging
+import hashlib
+from datetime import datetime, timedelta, timezone
+from flask import request, jsonify, current_app
+from functools import wraps
+from typing import Optional, Dict, Any, Tuple
```

Ruff (0.11.9)

`typing.Tuple` imported but unused

Remove unused import: `typing.Tuple`

(F401)